packages/material-ui/src/FieldTemplate/WrapIfAdditional.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+
+import { utils } from "@rjsf/core";
+import { JSONSchema7 } from "json-schema";
+
+import Grid from "@material-ui/core/Grid";
+import FormControl from "@material-ui/core/FormControl";
+import Input from "@material-ui/core/Input";
+import InputLabel from "@material-ui/core/InputLabel";
+
+import IconButton from "../IconButton/IconButton";
+
+const { ADDITIONAL_PROPERTY_FLAG } = utils;
+
+type WrapIfAdditionalProps = {
+  children: React.ReactElement;
+  classNames: string;
+  disabled: boolean;
+  id: string;
+  label: string;
+  onDropPropertyClick: (index: string) => (event?: any) => void;
+  onKeyChange: (index: string) => (event?: any) => void;
+  readonly: boolean;
+  required: boolean;
+  schema: JSONSchema7;
+};
+
+const WrapIfAdditional = ({
+  children,
+  classNames,
+  disabled,
+  id,
+  label,
+  onDropPropertyClick,
+  onKeyChange,
+  readonly,
+  required,
+  schema,
+}: WrapIfAdditionalProps) => {
+  const readonlyAsDisabled = true;
```

Comment:
I think this can be removed... Indeed that `antd` theme uses this alot, but it is taken from the `formContext`, not sure how it is relevant here, or how `formContext` can be applied to Material theme

Reply:
I agree, will update.